instagram/app/assets/stylesheets/photos.scss

```diff
@@ -0,0 +1,138 @@
+// Place all the styles related to the photos controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+h2 {
+	font-size: 16px;
+}
+
+#post {
+	border: 1px solid #dbc7c7;
```

Comment:
Color literals like `#dbc7c7` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented with spaces, not tabs<br>Properties should be ordered background, border, margin-bottom, padding-bottom